LayoutTests/platform/glib/TestExpectations

```diff
@@ -1821,6 +1821,8 @@ fast/mediastream/video-rotation.html [ Skip ]
 webrtc/video-rotation-no-cvo.html [ Failure Timeout ]
 webrtc/video-rotation-black.html [ Crash Failure Pass Timeout ]
 
+webrtc/video-clone-track.html [ Failure ]
+
```

Comment:
Please add a link to the glib equivalent bug

Reply:
Done